src/System.Text.Encoding/tests/Encoding/EncodingGetEncodingTests.cs

```diff
@@ -99,10 +100,10 @@ public void TestEncodingNameAndCopdepageNumber()
         [Fact]
         public void GetEncoding_EncodingName()
         {
-            CultureInfo originalUICulture = CultureInfo.CurrentUICulture;
-            try
+            // Workaround issue: UWP culture is process wide
+            RemoteInvoke(() =>
             {
-                CultureInfo.CurrentCulture = new CultureInfo("en-US");
```

Comment:
Why was this failing on a non-English system when the culture was being set to en-US?

Reply:
It was CurrentCulture but should be Current**UI**Culture. So I changed to Current**UI**Culture and added RemoteInvoke as Workaround UWP culture is process wide issue.

Reply:
Ok, thanks.